E3Next/E3.cs

```diff
@@ -264,7 +264,11 @@ private static void Init()
                 //setup is done after the settings are setup.
                 //as there is an order dependecy
                 Setup.Init();
-
+                foreach (var SendCommand in E3.CharacterSettings.Misc_CommandOnStartup)
+                {
+                    MQ.Write(SendCommand);
```

Comment:
can probably put some color on this and be a bit more descriptive.

Reply:
Updated